web/integration-core/src/jsMain/kotlin/androidx/compose/web/sample/Sample.kt

```diff
@@ -201,7 +201,7 @@ fun main() {
                 opacity(0.3)
             }
 
-            className(MyClassName) + hover() style {
+            className(MyClassName) + hover style {
```

Comment:
is it because `hover()` is deprecated?

Reply:
yep, this is the only reason - I we want to be conservative as possible I can just suppress the warning